selfhost/bound.wyv

```diff
@@ -1,6 +1,7 @@
 module bound
 
 import wyvern.collections.llist
+import wyvern.String
```

Comment:
Shouldn't be necessary; String is a built-in type, and this is actually the signature for a helper library for strings (should probably give it a different name, actually)